app/assets/javascripts/controllers/network_router/network_router_form_controller.js

```diff
@@ -41,6 +46,30 @@ ManageIQ.angular.app.controller('networkRouterFormController', ['$http', '$scope
     miqService.miqAjaxButton(url, $scope.networkRouterModel, { complete: false });
   };
 
+  $scope.addInterfaceClicked = function() {
+    miqService.sparkleOn();
+    var url = '/network_router/add_interface/' + networkRouterFormId + '?button=add';
+    miqService.miqAjaxButton(url, $scope.networkRouterModel, { complete: false });
+  };
+
+  $scope.removeInterfaceClicked = function() {
+    miqService.sparkleOn();
+    var url = '/network_router/remove_interface/' + networkRouterFormId + '?button=remove';
+    miqService.miqAjaxButton(url, $scope.networkRouterModel, { complete: false });
+  };
+
+  $scope.cancelAddInterfaceClicked = function() {
+    miqService.sparkleOn();
+    var url = '/network_router/add_interface/' + networkRouterFormId + '?button=cancel';
+    miqService.miqAjaxButton(url);
+  };
+
+  $scope.cancelRemoveInterfaceClicked = function() {
+    miqService.sparkleOn();
+    var url = '/network_router/remove_interface/' + networkRouterFormId + '?button=cancel';
+    miqService.miqAjaxButton(url);
+  };
+
```

Comment:
Ahh, it looks like this is not needed as a result of earlier refactoring. More specifically, this PR was based off the add/remove host functionality on HostAggregate, so this  came from cancelAddHostClicked and cancelRemoveHostClicked, both of which were originally called in the UI form with manually-created buttons but should have been removed when I created x_custom_form_buttons_angular.